NohanaImagePicker/AssetListSelectableDateSectionController.swift

```diff
@@ -378,6 +378,30 @@ extension AssetListSelectableDateSectionController: PHPhotoLibraryChangeObserver
     func photoLibraryDidChange(_ changeInstance: PHChange) {
         DispatchQueue.main.async { [weak self] in
             guard let self = self else { return }
+
+            let changeDetails = self.photoKitAssetList.changeDetails(changeInstance)
+            if let beforeResult = changeDetails?.fetchResultBeforeChanges,
+                let afterResult = changeDetails?.fetchResultAfterChanges {
+
+                var afterAssets: [PHAsset] = []
+                var removedAssets: [PHAsset] = []
+                for index in 0..<afterResult.count {
+                    let asset = afterResult.object(at: index)
+                    afterAssets.append(asset)
+                }
+
+                for index in 0..<beforeResult.count {
+                    let asset = beforeResult.object(at: index)
+                    if !afterAssets.map({ $0.localIdentifier }).contains(asset.localIdentifier) {
+                        removedAssets.append(asset)
+                    }
+                }
+
+                for removedAsset in removedAssets {
+                    self.nohanaImagePickerController.dropAsset(PhotoKitAsset(asset: removedAsset))
+                }
+            }
```

Comment:
We recommend syncing in the following ways.

Please refer 
```swift
            var removedAssets: [PhotoKitAsset] = []
            self.nohanaImagePickerController.pickedAssetList.forEach { item in
                if let asset = item as? PhotoKitAsset, let changeDetail = changeInstance.changeDetails(for: asset.asset) {
                    if changeDetail.objectWasDeleted {
                        removedAssets.append(asset)
                    }
                }
            }
            removedAssets.forEach { asset in
                self.nohanaImagePickerController.dropAsset(asset)
            }
            if !removedAssets.isEmpty {
                self.setToolbarTitle(self.nohanaImagePickerController)
            }
```

Reply:
I've modified this logic following your recommendation.
That seems to be better than before, moreover simple.

Reply:
Is toolbar title string update?
if it's not update, please run `setToolbarTitle()` function.